webui/react/src/pages/F_ExpList/glide-table/GlideTable.tsx

```diff
@@ -0,0 +1,563 @@
+      },
+      duration: {
+        id: 'duration',
+        isNumerical: true,
+        renderer: (record: ExperimentItem) => ({
+          allowOverlay: false,
+          data: `${getDurationInEnglish(record)}`,
+          displayData: `${getDurationInEnglish(record)}`,
+          kind: GridCellKind.Text,
+        }),
+        title: 'Duration',
+        width: columnWidths.duration,
+      },
+      forkedFrom: {
+        id: 'forkedFrom',
+        renderer: (record: ExperimentItem) => ({
+          allowOverlay: false,
+          copyData: String(record.forkedFrom ?? ''),
+          data: {
+            kind: 'links-cell',
+            links:
+              record.forkedFrom !== undefined
+                ? [
+                    {
+                      onClick: () =>
+                        record.forkedFrom && navigate(paths.experimentDetails(record.forkedFrom)),
+                      title: String(record.forkedFrom ?? ''),
+                    },
+                  ]
+                : [],
+            navigateOn: 'click',
+            underlineOffset: 6,
+          },
+          kind: GridCellKind.Custom,
+          readonly: true,
+        }),
+        title: 'Forked From',
+        width: columnWidths.forkedFrom,
+      },
+      id: {
+        id: 'id',
+        renderer: (record: ExperimentItem) => ({
+          allowOverlay: false,
+          copyData: String(record.id),
+          data: {
+            kind: 'links-cell',
+            links: [
+              {
+                onClick: () => navigate(paths.experimentDetails(record.id)),
+                title: String(record.id),
+              },
+            ],
+            navigateOn: 'click',
+            underlineOffset: 6,
+          },
+          kind: GridCellKind.Custom,
+          readonly: true,
+        }),
+        title: 'ID',
+        width: columnWidths.id,
+      },
+      name: {
+        id: 'name',
+        renderer: (record: ExperimentItem) => ({
+          allowOverlay: false,
+          copyData: String(record.name),
+          data: {
+            kind: 'links-cell',
+            links: [
+              {
+                onClick: () => navigate(paths.experimentDetails(record.id)),
+                title: String(record.name),
+              },
+            ],
+            navigateOn: 'click',
+            underlineOffset: 6,
+          },
+          kind: GridCellKind.Custom,
+          readonly: true,
+        }),
+        themeOverride: { horizontalBorderColor: '#225588' },
+        title: 'Name',
+        width: columnWidths.name,
+      },
+      numTrials: {
+        id: 'numTrials',
+        isNumerical: true,
+        renderer: (record: ExperimentItem) => ({
+          allowOverlay: false,
+          data: record.numTrials,
+          displayData: String(record.numTrials),
+          kind: GridCellKind.Number,
+        }),
+        title: 'Trials',
+        width: columnWidths.numTrials,
+      },
+      progress: {
```

Comment:
Is it possible to add bar frame like 
<img width="118" alt="image" src="https://user-images.githubusercontent.com/109113616/228915336-d4fd8899-c6a8-4fca-a5f4-3983890aad51.png">
 
and nice to show percentage by hovering on the bar